.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
I added this in because the "--api" tag that we used to create the repo removes it. I felt that I needed it to do adequate testing.

Reply:
nice work!